test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/CompatibilityIssueTests.cs

```diff
@@ -6,7 +6,7 @@
 using NuGet.Versioning;
 using Xunit;
 
-namespace NuGet.Commands.Test
+namespace NuGet.Commands.Test.RestoreCommandTests
```

Comment:
Is this stylistic only?

Reply:
Basically, yes.

Namespace hierarchy not matching the directory hierarchy really bugs me. My position is that if the files benefit from organization in subdirectories, so do namespaces. Plus VS creates files with namespaces matching the directory structure, so using anything different means one more (little) thing where we're fighting against the IDE.

Unfortunately, we have a class named `RestoreCommand`, in addition to a directory with the same name. So, using namespaces that match directory structure means that code gets confused when the namespace and class are "visible" at the same time, making it impossible to use the class, without also specifying the full namespace. In other words, `NuGet.Commands.RestoreCommand`, because otherwise using `RestoreCommand`, the compiler always thinks it's referring to `NuGet.Commands.Test.RestoreCommand` namespace, and even using `using RestoreCommand = NuGet.Commands.RestoreCommand` doesn't work.  Hence I renamed the directory to `RestoreCommandTests`, and updated all affected files to use the matching namespace.

Reply:
Good to know. 
FWIW, for QB mode bugs, the file count + line count tends to be noticed, so for future reference PRs intended to be patch, should be as minimal as can be :)